care/facility/api/viewsets/hospital_doctor.py

```
@@ -42,5 +43,11 @@ def get_facility(self):
             facility_qs.filter(users__id__exact=self.request.user.id)
         return get_object_or_404(facility_qs)
 
+    def list(self, request, *args, **kwargs):
+        response = super().list(request, *args, **kwargs)
+        total_doctors = self.get_queryset().aggregate(total_doctors=Sum('count'))['total_doctors']
```

Ruff (0.8.0)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)